funnel/views/commentvote.py

```diff
@@ -1,27 +1,50 @@
 from collections import namedtuple
 
-from flask import abort, flash, jsonify, redirect
+from flask import abort, flash, jsonify, redirect, request
 
 from baseframe import _, forms, request_is_xhr
+from baseframe.forms import Form, render_form
 from coaster.auth import current_auth
 from coaster.views import (
     ModelView,
     UrlForView,
+    render_with,
     requires_permission,
     requires_roles,
     route,
 )
 
 from .. import app, funnelapp
-from ..forms import CommentDeleteForm, CommentForm
-from ..models import Comment, Commentset, Proposal, db
+from ..forms import CommentForm
+from ..models import Comment, Commentset, Proposal, Voteset, db
 from .decorators import legacy_redirect
 from .login_session import requires_login
 from .mixins import ProposalViewMixin
 
 ProposalComment = namedtuple('ProposalComment', ['proposal', 'comment'])
 
 
+@Comment.views('url')
+def comment_url(obj):
+    url = None
+    commentset_url = obj.commentset.views.url()
+    if commentset_url is not None:
+        url = commentset_url + '#c' + obj.uuid_b58
```

Comment:
I'd like this to be `#c-` or `#c_`. We have to make this change in a few places around the code.